openGOALModLauncher.py

```diff
@@ -257,15 +260,14 @@ def getRefreshedTableData(sort_col_idx):
         #   mod["latest_available_update_date"] = f"{update_date:%Y-%m-%d %H:%M}"
         
         # only add to data if passes filter (if any)
-        if (
-            FILTER_STR is None
```

Comment:
think this would have always been showing jak 2 with no filter? or maybe FILTER_STR is always "" rather than `None`